src/components/state-details.js

```diff
@@ -126,14 +126,19 @@ export default function StateDetailsPage ({ location, data }) {
     Cars_All: carsAll,
     EV_Registration: evRegistration,
   } = data.allVehiclesJson.edges[0].node
+  
   const pctEv = Math.round((evRegistration / carsAll) * 100 * 10) / 10
   const pctNonEv = Math.round((100 - pctEv) * 10) / 10
+
+  // calculate cars remaining to electrify
+  const carsToElectrify = carsAll * (pctNonEv/100)
```

Comment:
We already have `evRegistration `, why not just do `carsAll - evRegistration`? Or you could do `carsAll * pctNonEv`

Reply:
sure - subtraction is easier. change made